### DISCUSSION TITLE
Let's work on a name for individual nodes of the Metaverse

### DISCUSSION THREAD

**Discussion by RedEagleP1** [omigroup/omigroup#382] [R&D]
The Facebook-Metaverse name change situation made it clear to us that as part of the open metaverse we need to be proactive to name the space in order to ensure it remains in a public and not private orbit. 

Shall we suggest and agree on a name for an individual node within the Metaverse? 

Like a webpage is part of the greater world wide web, let's name one node of the metaverse, come up with the definition and publish it together as OMI (and we at OCEM will help propagate it). 

We need to compromise to make this happen but it's a really small thing. So let's suggest names and let's agree on one together?

**Comment by mrmetaverse** [comment]
I love the concept of metaverse -> metagalaxy -> metaworld, etc. 

I also subscribe to the term Platform, as it still applies in 99% of the cases where someone says they are building _a_ metaverse.
